tools/c7n_gcp/c7n_gcp/query.py

```diff
@@ -101,39 +101,76 @@ def __init__(self, manager):
         self.manager = manager
 
     def get_resources(self, query):
-        session = local_session(self.manager.session_factory)
+        session = self.manager.session_factory(
+            quota_max_calls=300,
```

Comment:
I was curious where you got these numbers. based on [compute api limits ](https://cloud.google.com/compute/docs/api-rate-limits) it looks to be refreshed on a 100 second basis. Also should this be configurable by the user if they want more/less quota limits.  for example, to limit costs, or if they have increases api limits from google.

Reply:
rate limits are different for each service, for asset inventory the numbers are documented here
https://cloud.google.com/asset-inventory/docs/quota

360 per 1m per scope. we go to 300 here to preserve capacity for other api consumers.

Reply:
ah got it. thanks. didn't check it was just for the AssetInventory source.